doc/scapy/introduction.rst

```diff
@@ -6,84 +6,123 @@ Introduction
 
 About Scapy
 ===========
-Scapy is a Python program that enables the user to send, sniff and dissect and forge network packets. This capability allows construction of tools that can probe, scan or attack networks.
```

Comment:
Could you keep those on a single line ?
We know that it looks bad in the raw file, but `ReadTheDocs` automatically turns that into a paragraph.

The real upside is that having it stay on one line allows us to keep a clean git history, showing each change on the paragraph. If we were to split the line, the paragraph would be refactored with spacing fixes every time we make an edit.

Finally, it's very hard for us to properly review the PR :/